ui/guest/src/controls/rte.ts

```diff
@@ -42,23 +42,34 @@ export function initTinyMCE(
   const { field, model } = iceRegistry.getReferentialEntries(record.iceIds[0]);
   const type = field?.type;
   const inlineElsRegex =
-    /B|BIG|I|SMALL|TT|ABBR|ACRINYM|CITE|CODE|DFN|EM|KBD|STRONG|SAMP|VAR|A|BDO|BR|IMG|MAP|OBJECT|Q|SCRIPT|SPAN|SUB|SUP|BUTTON|INPUT|LABEL|SELECT|TEXTAREA/;
+    /^B$|^BIG$|^I$|^SMALL$|^TT$|^ABBR$|^ACRINYM$|^CITE$|^CODE$|^DFN$|^EM$|^KBD$|^STRONG$|^SAMP$|^VAR$|^A$|^BDO$|^BR$|^IMG$|^MAP$|^OBJECT$|^Q$|^SCRIPT$|^SPAN$|^SUB$|^SUP$|^BUTTON$|^INPUT$|^LABEL$|^SELECT$|^TEXTAREA$/;
```

Comment:
Was the regexp not matching?

Maybe we shouldn't be looking at elements but looking at their style property since one can change de display of it via styles. It says below the display prop doesn't matter, though, why?

Reply:
We used to change the display of the element, but there was an bug reported [here](https://github.com/craftercms/craftercms/issues/5212), and we ended up cloning the elements because it didn't matter the display style of the elements, the inline elements (the ones in the regex) all have the issue, and it doesn't happen to block type elements.

The regex was incorrectly matching, that's the reason I added the `^` and `$` to exactly match those words.

Reply:
What was it matching incorrectly?

Not sure about the performance but seems more readable to do: `/^(B|BIG|I|...)$/`. Please test.